### DISCUSSION TITLE
https://blog.tidy-intelligence.com/posts/dplyr-vs-polars/

### DISCUSSION THREAD

**Discussion by giscus** [tidy-intelligence/blog#6] [Announcements]
# https://blog.tidy-intelligence.com/posts/dplyr-vs-polars/

A comparison of R’s dplyr and Python’s polars data manipulation packages

https://blog.tidy-intelligence.com/posts/dplyr-vs-polars/

**Comment by kyleGrealis** [comment]
Thanks for writing this out. It's a great starting point for me transitioning to Python. Would you have any interest in doing more of a deep dive of `dplyr`-like functions for heavy data cleaning in Python? For example, chaining different methods together as would be done in a `dplyr` pipeline to make a nice, cleaned dataset.

**Reply by christophscheuch** [reply]
If you are looking for {dplyr}-like syntax in Python, I'd recommend {ibis-framework}. I use it for a real-world example here: https://blog.tidy-intelligence.com/posts/sakura-visualizations/.

I'm also working on a translation of this customer churn example to {ibis-framework} + {scikit-learn}, but I still need to iron out some issues: https://blog.tidy-intelligence.com/posts/classification-customer-churn/
